docs/community-clients/index.asciidoc

```diff
@@ -238,16 +238,19 @@ The following projects appear to be abandoned:
 * https://github.com/sksamuel/elastic4s[elastic4s]:
   Scala DSL.
 
-* https://github.com/scalastuff/esclient[esclient]:
-  Thin Scala client.
-
 * https://github.com/gphat/wabisabi[wabisabi]:
   Asynchronous REST API Scala client.
+  
+* https://github.com/workday/escalar[escalar]:
+  Type-safe Scala wrapper for the REST API.
 
 * https://github.com/SumoLogic/elasticsearch-client[elasticsearch-client]:
   Scala DSL that uses the REST API. Akka and AWS helpers included.
 
-The following project appears to be abandoned:
+The following projects appear to be abandoned:
+
+* https://github.com/scalastuff/esclient[esclient]:
+  Thin Scala client.
```

Comment:
The last commit on this project was ~2.5 years ago, so thought it made sense to move it to this section